app/views/layouts/_top_bar.html.erb

```diff
@@ -53,7 +53,7 @@
           <% if user_signed_in? %>
             <span class="crayons-avatar crayons-avatar--l"><img class="crayons-avatar__image" alt="" id="nav-profile-image" width="32" height="32" /></span>
           <% else %>
-            <%= inline_svg_tag("hamburger.svg", aria: true, class: "crayons-icon", title: "Navigation menu") %>
+            <%= inline_svg_tag("hamburger.svg", aria: false, class: "crayons-icon") %>
```

Comment:
```suggestion
            <%= inline_svg_tag("hamburger.svg", class: "crayons-icon") %>
```

Reply:
@ludwiczakpawel, I'm guessing you're suggesting to remove it because the [default is false](https://github.com/jamesmartin/inline_svg#accessibility)?

Reply:
That's how I understood it, thought I made a commit for it but maybe there was more than one in the PR. Thanks!